test/data.jl

```diff
@@ -271,13 +272,6 @@ module TestData
         v1 = randn(10)
     )
 
-    dt2 = DataTable(
-        a = [:x,:y][[1,2,1,1,2]],
-        b = [:A,:B,:C][[1,1,1,2,3]],
-        v2 = randn(5)
-    )
-    dt2[1,:a] = Nullable()
```

Comment:
Automatic promotion could be problematic as it means you could get missing values in a column where you wouldn't have expected them. Probably better require explicit conversion.Automatic promotion could be problematic as it means you could get missing values in a column where you wouldn't have expected them. Probably better require explicit conversion.